maas-schemas/schemas/maas-backend/subscriptions/subscription-intent.json

```diff
@@ -11,7 +11,7 @@
         "planAddons": {
           "type": "array",
           "items": {
-            "$ref": "#/definitions/addonId"
+            "$ref": "http://maasglobal.com/maas-backend/subscriptions/subscription.json#/definitions/addon"
```

Comment:
Note for myself. Same change should be applied to `coupons`.

Reply:
@thanhtr This is WIP but please take a look at this. IMO this is required. It covers the scenario where you want to add U30 with HSL AB ticket and then the addon has unitPrice of 0 EUR.
Otherwise, if I just use plan ID and addon ID I see charges basically doubled (62,70 EUR for plan `fi-whim-urban-freedom-30-day_v2` and 59.70 EUR for `fi-hsl-ab-30-day_v1` addon).